dev-docs/adunit-reference.md

```diff
@@ -0,0 +1,414 @@
+| `ratio_width`  | Required | Integer | See above.                                                                                           |
+
+<a name="adUnit.mediaTypes.video" />
+
+#### adUnit.mediaTypes.video
+
+{: .table .table-bordered .table-striped }
+| Name             | Scope       | Type                   | Description                                                                                                                                                         |
+|------------------+-------------+------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `context`        | Optional    | String                 | The video context, either `"instream"` or `"outstream"`.  Example: `context: "outstream"`                                                                           |
+| `playerSize`     | Optional    | Array[Integer,Integer] | The size (width, height) of the video player on the page, in pixels.  Example: `playerSize: [640, 480]`                                                             |
+| `mimes`          | Recommended | Array[String]          | Content MIME types supported, e.g., `"video/x-ms-wmv"`, `"video/mp4"`. **Required by OpenRTB when using [Prebid Server][pbServer]**.                                |
+| `protocols`      | Optional    | Array[Integer]         | Array of supported video protocols.  For list, see [OpenRTB spec][openRTB]. **Required by OpenRTB when using [Prebid Server][pbServer]**.                           |
+| `playbackmethod` | Optional    | Array[Integer]         | Allowed playback methods. If none specified, all are allowed.  For list, see [OpenRTB spec][openRTB]. **Required by OpenRTB when using [Prebid Server][pbServer]**. |
+
+<a name="adUnit-examples" />
+
+## Examples
+
++ [Banner](#adUnit-banner-example)
++ [Video](#adUnit-video-example)
++ [Native](#adUnit-native-example)
++ [Multi-Format](#adUnit-multi-format-example)
+
+<a name="adUnit-banner-example">
+
+### Banner
+
+For an example of a banner ad unit, see below.  For more detailed instructions, see [Getting Started]({{site.baseurl}}/dev-docs/getting-started.html).
+
+```javascript
+pbjs.addAdUnits({
+    code: slot.code,
+    mediaTypes: {
+        banner: {
+            sizes: [[300, 250], [300, 600]]
+        },
+        bids: [{
+            bidder: 'appnexus',
+            params: {
+                placementId: '9880618'
+            }
+        }, ]
+    }
+})
+```
+
+<a name="adUnit-video-example">
+
+### Video
+
+For an example of an instream video ad unit, see below.  For more detailed instructions, see [Show Video Ads]({{site.baseurl}}/dev-docs/show-video-with-a-dfp-video-tag.html).
+
+```javascript
+pbjs.addAdUnits({
+    code: 'video',
+    mediaTypes: {
+        video: {
+            context: "instream",
+            sizes: [640, 480],
```

Comment:
Should be "playerSize" instead of "sizes".

Reply:
Addressed in 0333fd6